sites/next.skeleton.dev/src/components/docs/NavGrid.astro

```diff
@@ -1,22 +1,28 @@
 ---
-import { getCollection } from 'astro:content';
+import { getCollection, type CollectionEntry } from 'astro:content';
+
+interface Props {
+	collection: 'docs';
+	path: string;
+	classes: string;
+}
 
 const { collection, path, classes } = Astro.props;
 
 // Get collection, filter by path
-const collectionItems = await getCollection(collection, (item: any) => {
+const collectionItems = await getCollection(collection, (item) => {
 	if (item.slug.includes(path) && !item.id.endsWith('meta.mdx')) return item;
 });
 
 // Sort Order
-const sortByOrder = (a: any, b: any) => {
+const sortByOrder = (a: CollectionEntry<'docs'>, b: CollectionEntry<'docs'>) => {
```

Comment:
Not an issue - just stating that all typing adjustments appears to be working fine on my machine. Great job!